src/tests/test_dte_data_models.py

```diff
@@ -1579,24 +1722,27 @@ def test_validate_referencias_rut_otro_is_consistent_with_tipo_dte(self) -> None
 
         expected_validation_errors = [
             {
-                'loc': ('__root__',),
-                'msg': "Setting a 'rut_otro' is not a valid option for this 'tipo_dte':"
-                " 'tipo_dte' == <TipoDte.FACTURA_ELECTRONICA: 33>,"
-                " 'Referencia' number 1.",
+                'loc': (),
+                'msg': "Value error, Setting a 'rut_otro' is not a valid option for this "
+                "'tipo_dte': 'tipo_dte' == <TipoDte.FACTURA_ELECTRONICA: 33>, "
+                "'Referencia' number 1.",
```

Comment:
Reformat to reduce diff size.

```suggestion
                'msg': (
                    "Value error, "
                    "Setting a 'rut_otro' is not a valid option for this 'tipo_dte':"
                    " 'tipo_dte' == <TipoDte.FACTURA_ELECTRONICA: 33>,"
                    " 'Referencia' number 1."
                ),
```